docs/quick-start.md

```diff
@@ -421,16 +421,15 @@ $ YOUR_SPARK_HOME/bin/spark-submit \
 Lines with a: 46, Lines with b: 23
 {% endhighlight %}
 
-If you have PySpark pip installed into your enviroment (e.g. `pip instal pyspark` you can run your application with the regular Python interpeter or use the provided spark-submit as you prefer.
+If you have PySpark pip installed into your enviroment (e.g., `pip install pyspark` you can run your application with the regular Python interpreter or use the provided 'spark-submit' as you prefer.
```

Comment:
others look good but for the last one ...

`` pip install pyspark` `` ->  `` pip install pyspark`) ``  (missing closing parenthesis).